code/stable/glassbr/SRS/JSON/GlassBR_SRS.ipynb

```diff
@@ -111,7 +111,7 @@
     "|$J$|Stress distribution factor (Function)|--|\n",
     "|$J_max$|Maximum value for the stress distribution factor|--|\n",
     "|$J_min$|Minimum value for the stress distribution factor|--|\n",
-    "|$J_tol$|Stress distribution factor (Function) based on Pbtol|--|\n",
+    "|$J_tol$|Stress distribution factor (Function)|--|\n",
```

Comment:
Now we have two values with the description "Stress distribution factor (Function)"; it's (probably) obviously not correct, but are we OK with this for now?

Reply:
Good catch @samm82.  By removing the Pbtol, we removed the information that J_tol is a specific value of J.  @balacij can you change the definition of J_tol to "Tolerable stress distribution factor"?

Reply:
Yep! Updated it now.